README.md

```diff
@@ -5,16 +5,56 @@
 [![Clojars Project](https://img.shields.io/clojars/v/banach.svg)](https://clojars.org/banach)
 
 
-
 This contains utilities that can be useful when working with [Manifold](https://github.com/ztellman/manifold).
 
-At the moment the library provides a way to create retry functions for deferred
-producing operations.
+## Retries
+
+Banach provides flexible tools for doing retries of functions that can fail. By
+default, it offers composable exponential backoff, maximum number of attempts,
+and hard failures (that stop future retry attempts).
+
+Minimum viable snippet (with the `banach.retry` namespace aliased as `r`):
+
+```clojure
+(r/retry f (comp (r/exponentially 2) (r/up-to 5)))
+```
```

Comment:
Am I going blind? Did I mess up the backticks? Did GitHub highlighting get really confused and start highlighting the next few paragraphs as Clojure?